installer/test/CreateInstallerTestPackages/Makefile.mak

```diff
@@ -21,7 +21,7 @@ SIGNTOOL_OPTS=/v
 TARGET_BASENAME=Microsoft.WindowsAppRuntime.Test.InstallFramework
 
 TargetDir=$(OutDir)$(TargetName)
-MSTestPfx=$(SolutionDir)..\..\..\temp\MSTest.pfx
+WinAppSDKTestPfx=$(SolutionDir)..\..\..\.user\winappsdk.certificate.test.pfx
```

Comment:
Do not use SolutionDir.  Use RepoRoot instead with full paths.

Reply:
That's going away. I haven't updated installer\** yet, but I didn't want to wait until then before getting feedback. That's just yet-more-of-the-same convert older to MakeMsix.targets.